src/tests/example.spec.js

```diff
@@ -28,3 +24,80 @@ test.describe('Saucedemo app basic tests', () => {
         await expect(app.shoppingCart.cartItems).not.toBeAttached();
     });
 });
+
+
+
+test.describe('Saucedemo Unit 10 tests', () => {    
```

Comment:
![image](https://github.com/user-attachments/assets/ec1d3c42-e55e-4e01-9aa1-6cd637d17ab8)
you should definitely try parametrized tests

Reply:
https://playwright.dev/docs/test-parameterize